packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -49,9 +49,30 @@ exports.sourceNodes = async (
     disallowedLinkTypes,
     skipFileDownloads,
     fastBuilds,
+    translation,
   } = pluginOptions
   const { createNode, setPluginStatus, touchNode } = actions
 
+  // Default apiBase to `jsonapi`
+  apiBase = apiBase || `jsonapi`
+
+  // Default disallowedLinkTypes to self, describedby.
+  disallowedLinkTypes = disallowedLinkTypes || [`self`, `describedby`]
+
+  // Default concurrentFileRequests to `20`
+  concurrentFileRequests = concurrentFileRequests || 20
+
+  // Default skipFileDownloads to false.
+  skipFileDownloads = skipFileDownloads || false
+
+  // Determine what entities can be translated and what languages are enabled.
+  
+  const languageConfig = (
+    store.getState().status.plugins?.[`gatsby-source-drupal`]?.languageConfig
+    || await fetchLanguageConfig({ translation, baseUrl, apiBase, basicAuth, headers, params })
+  )
+  setPluginStatus({ languageConfig })
```

Comment:
I'm not clear enough on the Gatsby API `setPluginStatus` to know whether this will overwrite the other keys in the status? Should I deconstruct existing params into this object or will it do that form me?

Reply:
Further testing seems to prove that this won't affect other status values.